src/app/core/models/user.model.ts

```diff
@@ -1,6 +1,13 @@
 export interface User {
-    body: string,
     id:number,
-    title:string,
-    userId: number
+    name: string,
+    email: string,
+    mobile?: string,
+    city?: string,
+    age?: string,
+    memberOf?: string,
```

Comment:
I think it would be better if "memberOf " has type Array and age has type number?

Reply:
age type number - ok. memberOf - let's stay as it is. If we need change we will change.